src/Microsoft.ML.CodeGenerator/CodeGenerator/CSharp/TransformGenerators.cs

```diff
@@ -330,4 +330,21 @@ public override string GenerateTransformer()
         }
     }
 
+    internal class ImageLoading : TransformGeneratorBase
+    {
+        public ImageLoading(PipelineNode node) : base(node)
+        {
+        }
+
+        internal override string MethodName => "LoadRawImageBytes";
+
+        public override string GenerateTransformer()
+        {
+            string inputColumn = InputColumns.Count() == 1 ? InputColumns[0] : throw new Exception($"output columns for the suggested transform: {MethodName} are null");
```

Comment:
This exception message doesn't make sense for the check it is performing. If the `InputColumns.Count == 2` the message will say $"output columns for the suggested transform: {MethodName} are null".

1. "Input" instead of "output".
2. "are null" doesn't make sense when there are 2 of them.